rover/src/main.c

```diff
@@ -72,11 +81,23 @@ void right();
 void read_us();
 void read_ir();
 void run();
+void us_led();
+void pixel();
+void uart_out();
 uint32_t get_us(uint32_t trig, uint32_t echo, struct device *dev);
 
 void main(void)
 {
-	int ret;
+	k_sleep(2000);
```

Comment:
Any reason to have a sleep here? If so, please add a comment justifying!

Reply:
arduino takes time to initialize.